Directory.Packages.props

```diff
@@ -57,5 +58,6 @@
     <PackageVersion Include="xunit.extensibility.execution" Version="2.4.2" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.5" />
     <PackageVersion Include="xunit" Version="2.4.2" />
+    <PackageVersion Include="Grpc.AspNetCore" Version="2.40.0" />
```

Comment:
This entry is already there in the file

Reply:
@agyen you only need ` <PackageVersion Include="Grpc.AspNetCore" Version="2.62.0" />` and it's already there on line 14. 
Please remove line 61.

Reply:
It breaks the build when I do that

Reply:
I see. it's possible that the implementation relies on Grpc.AspNetCore 2.40.0. There could be breaking changes between 2.62.0 and 2.40.0

Reply:
The build is successful now with Grpc.AspNetCore 2.62.0.